simulations/validator_client.py

```diff
@@ -0,0 +1,32 @@
+from utils.clock import Clock
+
+
+class ValidatorClient(Clock):
```

Comment:
Do we want a ValidatorClient to descend from a clock, or create a clock when it's created? I'm leaning more to the latter, but let me know your thoughts.

Reply:
Not inheriting from Clock in other PR.
Bringing the PRs together with notes shortly